user-docs/latest/users-roles-permissions/configuring-administrator-roles.md

```diff
@@ -0,0 +1,160 @@
+# Configuring administrator roles
+
+Administrators are the users of an admin panel of a Strapi application. Administrator accounts and roles are managed with the Role-Based Access Control (RBAC) feature. It is available in the *Administration panel* category of the Settings interface, accessible from *General > Settings* in the main navigation of the admin panel.
+
+The *Administration panel* category is divided into 2 sub-categories: *Roles* and *Users* (see [Managing administrators](managing-administrators.md)).
+
+![Administrator roles interface](../assets/users-permissions/administrator_roles.png)
+
+The *Roles* sub-category of *Administration panel* displays all created roles for the administrators of your Strapi application.
+
+From this interface, it is possible to:
+
+- create a new administrator role (see [Creating a new role](#creating-a-new-role)),
+- delete an administrator role (see [Deleting a role](#deleting-a-role)),
+- or access information regarding an administrator role, and edit it (see [Editing a role](#editing-a-role)).
+
+By default, 3 administrator roles are defined for any Strapi application:
+
+- Author: to be able to create, manage and publish their own content.
```

Comment:
```diff
- - Author: to be able to create, manage and publish their own content.
+ - Author: to be able to create and manage their own content.
```
In the Author role, the publish permission check box doesn't exist.
Only Editor can publish.